packages/web-pkg/src/composables/piniaStores/extensionRegistry.ts

```diff
@@ -1,15 +1,18 @@
 import { Action } from '../actions'
```

Comment:
I'd be a little more generic with the naming here, for example, if we replace Pinia store with something new in the future. Just in case

Reply:
We intentionally named it like this to have a clear separation between our current vuex store implementation and the pinia one. Once we moved to pinia I agree though 👍